packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -98,13 +99,20 @@ class WebFs {
     await _dwds.stop();
     await _server.close(force: true);
     await _chrome.close();
+    await _connectedApps?.cancel();
   }
 
   /// Retrieve the [DebugConnection] for the current application.
   Future<DebugConnection> runAndDebug() async {
```

Comment:
nit: async not needed.

Reply:
I'll remove in https://github.com/flutter/flutter/pull/39951